_dev/exist.md

```diff
@@ -98,6 +98,7 @@ visit
 
 **Footnotes**
 
-[1] The preview releases are usually named with a -RC suffix, like
-5.0.0-RC1. They *should* work fine, but may cause compatibility
-problems. Stick to the most recent, non-preview version.
+[^1]:
+    The preview releases are usually named with a -RC suffix, like 
+    5.0.0-RC1. They *should* work fine, but may cause compatibility 
+    problems. Stick to the most recent, non-preview version.
```

Comment:
Our servers are running eXist version 4. People should probably stick to the most recent release in the 4.x line.

Reply:
Changed this to 4 now.